notifications_utils/template.py

```diff
@@ -168,12 +169,15 @@ def content_count(self):
             # we always want to call SMSMessageTemplate.__str__ regardless of subclass, to avoid any html formatting
             SMSMessageTemplate.__str__(self)
             if self._values
-            else gsm_encode(add_prefix(self.content.strip(), self.prefix))
+            else gsm_and_welsh_encode(add_prefix(self.content.strip(), self.prefix))
         ).encode(self.encoding))
 
     @property
     def fragment_count(self):
-        return get_sms_fragment_count(self.content_count)
+        content_with_placeholders = str(self)
+        unicode = is_unicode(content_with_placeholders)
```

Comment:
`unicode` is a built-in, it's good practice not to overwrite it. i'd probably just in-line this into the `get_sms_fragment_count` call.

Reply:
I've changed this and the other place I'd used `unicode`